pkg/prettyerr/error.go

```diff
@@ -74,3 +74,18 @@ func Prettify(err error, opts ...StackOptionFn) (Errors, error) {
 
 	return stack, nil
 }
+
+// PrettyError prettifies error instance
+func PrettyError(err error) error {
```

Comment:
Please try to avoid stutter in code. the package is already called `prettyerr` I think it would be a good alternative to rename this method to `StandardErr`.